packaging/rpm/make-microshift-app-images-rpm.sh

```diff
@@ -69,7 +135,9 @@ sudo chmod -R a+rx  %{imageStore}
 
 
 %post
-sudo sed -i '/^additionalimagestores =*/a "$_img_dir_",' /etc/containers/storage.conf
+if ! grep -q "$IMG_DIR" /etc/containers/storage.conf; then
+    sudo sed -i '/^additionalimagestores =*/a "$IMG_DIR",' /etc/containers/storage.conf
```

Comment:
the other spec has a more elaborated version, it has a couple of details we could include here: 
```
%post

mkdir -p %{imageStore}
cd %{imageStore}
tar xvfjp %{tmpStore}/microshift-containers.tar.bz2
truncate -s 0 %{tmpStore}/microshift-containers.tar.bz

# only on install (1), not on upgrades (2)
if [ $1 -eq 1 ]; then
   sed -i '/^additionalimagestores =*/a "%{imageStore}",' /etc/containers/storage.conf
   # if crio was already started, restart it so it read from new imagestore
   systemctl is-active --quiet crio && systemctl restart --quiet crio
fi

%postun

# only on uninstall (0), not on upgrades(1)
if [ $1 -eq 0 ];
  sed -i '/"${imageStoreSed}",/d" /etc/containers/storage.conf
  systemctl is-active --quiet crio && systemctl restart --quiet crio

fi
```

Reply:
the removal sed part is broken on the example I posted.

Reply:
Got the delete sed sorted and added a noop on the crio check in case it was inactive to avoid scriplet failure errors. `/etc/containers/storage` inserts the filepath on install [1] and deletes it on [0] for uninstall.

Below is the output of the build/install/delete. Ty!

```sh
sudo ./make-rpm.sh   --image-list ./images.txt   --container-dir /var/lib/containers/storage/overlay-images   --rpmbuild-dir /home/brent/rpmbuild
[truncated]
Wrote: /home/brent/rpmbuild/RPMS/x86_64/microshift-app-images-1-1.x86_64.rpm
Executing(%clean): /bin/sh -e /var/tmp/rpm-tmp.TsDj1b
+ umask 022
+ cd /home/brent/rpmbuild/BUILD
+ /usr/bin/rm -rf /home/brent/rpmbuild/BUILDROOT/microshift-app-images-1-1.x86_64
+ exit 0
 
brent@rhel8-80:~/rpm-builds$ sudo rpm -ivh /home/brent/rpmbuild/RPMS/x86_64/microshift-app-images-1-1.x86_64.rpm
Verifying...                          ################################# [100%]
Preparing...                          ################################# [100%]
Updating / installing...
   1:microshift-app-images-1-1        ################################# [100%]
brent@rhel8-80:~/rpm-builds$ sudo podman image ls
REPOSITORY                    TAG         IMAGE ID      CREATED     SIZE        R/O
mirror.gcr.io/library/alpine  latest      0ac33e5f5afa  8 days ago  5.86 MB     true
brent@rhel8-80:~/rpm-builds$ sudo rpm -ev microshift-app-images
Preparing packages...
microshift-app-images-1-1.x86_64
brent@rhel8-80:~/rpm-builds$ sudo podman image ls
REPOSITORY  TAG         IMAGE ID    CREATED     SIZE
```